src/wallet/walletdb.h

```diff
@@ -43,18 +43,18 @@ struct WalletContext;
 static const bool DEFAULT_FLUSHWALLET = true;
 
 /** Error statuses for the wallet database */
-enum class DBErrors
+enum class DBErrors : int
```

Comment:
In commit "walletdb: Refactor legacy wallet record loading into its own function" (8648511b567dfcdea7ffa5ac4595a43a768f5525)

Would be helpful to have a code comment here saying enum number values are significant because if there are different error codes from reading different rows of the database, the error code with the highest number is the one that will be returned.

Reply:
Added a comment.